src/block/icon-label/schema.js

```diff
@@ -37,6 +37,11 @@ export const attributes = ( version = VERSION ) => {
 				stkResponsive: true,
 				default: '',
 			},
+			iconGap2: {
```

Comment:
Add a comment on `iconGap` that it's deprecated but kept here for migration purposes